seed/data/user/users.js

```diff
@@ -0,0 +1,20 @@
+module.exports = [
```

Comment:
Not sure how this file made it back into this branch. I thought we had deleted it as part of a different ticket a while back. This file needs to be deleted.

Reply:
done